content/docs/start/experiments.md

```diff
@@ -43,71 +42,87 @@ evaluate:
+[`precision_recall_curve`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_recall_curve.html))
+into plots file `prc.json`:
 
 ```json
 {
   "prc": [
     { "precision": 0.021473008227975116, "recall": 1.0, "threshold": 0.0 },
-    ...{ "precision": 1.0, "recall": 0.009345794392523364, "threshold": 0.64 }
+    ...
+    { "precision": 1.0, "recall": 0.009345794392523364, "threshold": 0.64 }
   ]
 }
 ```
 
 > DVC doesn't force you to use any specific file names, or even format or
-> structure of a metrics or plots file - it's pretty much user and case defined,
-> please, refer to `dvc metrics` and `dvc plots` documentation for more details.
+> structure of a metrics or plots file - it's pretty much user and case defined.
+> Please refer to `dvc metrics` and `dvc plots` for more details.
 
-Let's save this experiment to compare it later using DVC comparison commands:
+Let's save this experiment, so we can compare it later:
 
 ```dvc
 $ git add scores.json prc.json
 $ git commit -a -m "Create evaluation stage"
 ```
 
-We've managed to capture metrics that experiment has, and later we will see how
-this information can be used to compare different experiment iterations,
-visualize them (e.g. `ROC` or model loss plots). Let's see how can we capture
-another important piece of information that will be important to compare
-experiments - experiment parameters.
+Later we we will see how these and other can be used to compare and visualize
+different experiment iterations. For now, let's see how can we capture another
+important piece of information that will be useful to compare experiments:
+parameters.
 
 ## Defining parameters
 
-It's pretty common for data processing pipelines to use a separate YAML or JSON
-config file that defines adjustable parameters you use to train a model, do
-pre-processing, etc. DVC provides a mechanism for a stage to depend on values
-from such a config file. That's how stage is defined in the `dvc.yaml` file:
+It's pretty common for data science pipelines to include configuration files
+that defines adjustable parameters to train a model, do pre-processing, etc. DVC
+provides a mechanism for stages to depend on the values of specific sections of
+such a config file (YAML or JSON formats are supported).
+
+Luckily, we should already have a stage with
+[parameters](/doc/command-reference/params) in `dvc.yaml`:
+
+```yaml
+featurize:
+  cmd: python src/featurization.py data/prepared data/features
+  deps:
+    - data/prepared
+    - src/featurization.py
+  params:
+    - featurize.max_features
+    - featurize.ngrams
+  outs:
+    - data/features
+```
 
 <details>
 
-### 💡 Expand to recall how it was generated
+### 💡 Expand to recall how it was generated.
 
-As a reminder, the `featurize` stage was created with this command. No need to
-run it again, but pay more attention to the `-p` option this time:
+The `featurize` stage
```

Comment:
here as well ... it's not CR, informal language is fine ... I wish we had even more of this

Reply:
I think I only removed "As a reminder" here, because you mentioned the different Get Started pages are not supposed to be connected to each other necessarily. Instead I linked to that page. "pay more attention" sounds strange to me, like we're telling the user they're not paying enough attention 😋 (angry teacher).